app/borrowers/server.py

```diff
@@ -0,0 +1,27 @@
+from flask import jsonify
+
+
+def register_routes(blueprint):
+    @blueprint.route('/get-borrower/<borrower_id>', methods=['GET'])
+    def get_borrower(borrower_id):
+
+        if int(borrower_id) == 1:
+            result = {
+                "borrower_forename": "Peter",
```

Comment:
can we get rid of `borrower_` from the keys? by definition we are talking about borrowers, right?